assets/html/scss/documenter/components/_clipboard.scss

```diff
@@ -0,0 +1,3 @@
+.copy-button {
+  float: right;
+}
```

Comment:
Perhaps it would be good to make it a bit smaller?

```suggestion
  font-size: 0.9em;
}
```

I.e. this

![smaller](https://user-images.githubusercontent.com/147757/97771094-854b0100-1b9e-11eb-8276-010c984f364c.png)

instead of this

![bigger](https://user-images.githubusercontent.com/147757/97771099-8a0fb500-1b9e-11eb-9ec9-c6bb935df352.png)